src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyTextFilter.java

```diff
@@ -0,0 +1,650 @@
+ *   &lt;/module&gt;
+ *   &lt;module name=&quot;TreeWalker&quot;&gt;
+ *     &lt;module name=&quot;ConstantName&quot;&gt;
+ *       &lt;property name=&quot;id&quot; value=&quot;ignoreMe&quot;/&gt;
+ *     &lt;/module&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * static final int UPPER_CASE_CONSTANT1234567; // violation
+ * static final int lowerCaseConstant3;
+ * static final int UPPER_CASE_CONSTANT4567891011; // violation
+ * </pre>
+ * <p>
+ * To configure the filter to suppress audit events for the current and next 3 lines:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;Checker&quot;&gt;
+ *   &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+ *     &lt;property name=&quot;rowInfluence&quot; value=&quot;3&quot;/&gt;
+ *   &lt;/module&gt;
+ *   &lt;module name=&quot;TreeWalker&quot;&gt;
+ *     &lt;module name=&quot;ConstantNameCheck&quot;/&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * static final int lowerCaseConstant1; // SUPPRESS CHECKSTYLE the reason for this is..
+ * static final int lowerCaseConstant2;
+ * static final int lowerCaseConstant3;
+ * static final int lowerCaseConstant4;
+ * static final int lowerCaseConstant5; // violation
+ * </pre>
+ * <p>
+ * To configure the filter to suppress audit events for the current and previous 2 lines:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;Checker&quot;&gt;
+ *   &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+ *     &lt;property name=&quot;rowInfluence&quot; value=&quot;-2&quot;/&gt;
+ *   &lt;/module&gt;
+ *   &lt;module name=&quot;TreeWalker&quot;&gt;
+ *     &lt;module name=&quot;ConstantNameCheck&quot;/&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * static final int lowerCaseConstant; // violation
+ * static final int lowerCaseConstant1;
+ * static final int lowerCaseConstant2;
+ * static final int lowerCaseConstant3; // SUPPRESS CHECKSTYLE the reason for this is..
+ * static final int lowerCaseConstant4; // violation
+ * </pre>
+ * <p>
+ * To configure the filter to accept variable {@code checkPattern} and {@code rowInfluence}:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;Checker&quot;&gt;
+ *   &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+ *     &lt;property name=&quot;nearbyTextPattern&quot;
+ *       value=&quot;SUPPRESS CHECKSTYLE (\w+) - ([+-]\d+) Lines&quot;/&gt;
+ *     &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;$1&quot;/&gt;
+ *     &lt;property name=&quot;rowInfluence&quot; value=&quot;$2&quot;/&gt;
+ *   &lt;/module&gt;
+ *   &lt;module name=&quot;TreeWalker&quot;&gt;
+ *     &lt;module name=&quot;ConstantNameCheck&quot;/&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * static final int lowerCaseConstant1; // SUPPRESS CHECKSTYLE FileTabCharacterCheck - +3 Lines
```

Comment:
You meant ConstantName.

Examples are very long...
What if we use check does not need such long line and be easy to understand where is violation.
MagicNumber or AvoidStarImport.
```
import my.co.*; // violation 
```

Or 
```
int v = 45; // violation 
```

With such short in length of line example, there will be no much room to such mistakes and easy for users

Reply:
I will change over to `MagicNumber`, no complaints here.

Reply:
Done.